source/MRMesh/MRCircleObject.h

```diff
@@ -54,10 +65,14 @@ class MRMESH_CLASS CircleObject : public ObjectLinesHolder
     MRMESH_API virtual void serializeFields_( Json::Value& root ) const override;
 
     virtual Expected<std::future<VoidOrErrStr>> serializeModel_( const std::filesystem::path& ) const override
-        { return {}; }
+    {
+        return {};
+    }
 
     virtual VoidOrErrStr deserializeModel_( const std::filesystem::path&, ProgressCallback ) override
-        { return {}; }
+    {
+        return {};
+    }
```

Comment:
Can we avoid change of format in this file?

Reply:
done